zerver/openapi/zulip.yaml

```diff
@@ -9203,7 +9213,7 @@ paths:
                           details on what each feature level means.
 
                           **Changes**.  New in Zulip 3.0.  We recommend using an implied value
-                          of 0 for Zulip servers that do not send this field.
+                          of 0 for Zulip servers that do not send this field(feature level 1).
```

Comment:
We should always spell it as `Zulip x.y (feature level z)`, for consistency.  

We may in the future want to consider doing a macro or something for the syntax, which would also linkify these.